dotcom-rendering/src/types/request.ts

```diff
@@ -0,0 +1,3 @@
+export type BaseRequestBody = {
```

Comment:
Another way I have seen this named as it's not a type is `WithPageId` and then you ahve something like `type FEArticleType extends WithPageId`.

The guard mentioned above could fit in nicely here.